src/core/infrastructure/model.d.ts

```diff
@@ -34,7 +34,7 @@ import {Event} from './event';
 export declare class Model {
 	constructor();
 
-	static register(name: string, model: Model): Model;
+	static register(name: string, model: {new(): {}}): Model;
```

Comment:
What is the purpose of new()?

Reply:
It means, that the second arg is a type that has parameterless constructor

Reply:
@vkorolev thanks